base/setup/usetup/mui.c

```diff
@@ -30,6 +30,8 @@
 #define NDEBUG
 #include <debug.h>
 
+WORD SelectedCodePage = 437;
```

Comment:
The idea of this PR looks nice, but wouldn't it be possible instead to have those character translation (incl. MUITranslateChar) in the console.c code, where the console functions could adjust the characters being emitted to the video (this would be helpful for when also we redirect to e.g. a serial port).

Reply:
Added global variables for special characters.